frontend/scripts/metrica.js

```diff
@@ -9,12 +9,40 @@
             });
         }
     }
+
+    window.onerror = (errorMsg, errorUrl) => {
```

Comment:
I am not sure, but probably addEventListener better, in order not to be changed by user

And fifth parameter is error object, and you can take error.stack from here, because line number is not exactly what you need

https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror